PCG/pcg.py

```diff
@@ -144,17 +159,30 @@ def get_xy(self, index):
         self.unit_location_records.append((x, y))
         return x, y
 
-    def get_map(self):
+    def get_map(self, mapKey):
         root = ET.Element("rts.PhysicalGameState", width=str(self.width), height=str(self.height))
         self.initiate_terrain(root, "terrain", self.wallRings)
         self.initiate_players(root, "players")
         self.initiate_units(root, "units")
         tree = ET.ElementTree(root)
-        tree.write("./maps/filename.xml")
+        tree.write(os.path.join("PCG/maps/", "pcg_map" + "_" + str(mapKey) + ".xml"))
```

Comment:
Nit: the output directory should be configurable (not hard coded like this).

Reply:
So what kind of configurable output directory you are expecting?